src/main/webapp/WEB-INF/jsp/viewBookmarks.jsp

```diff
@@ -95,15 +95,15 @@
         </c:choose>
         <br>
         <c:if test="${!guestMode }">
-            <button type="button" onclick="newBookmark('${portletNamespace}');return false;" class="btn btn-default" style="${ guestMode ? 'display: none;' : '' }"><spring:message code="portlet.view.addBookmark"/></button>
+            <button type="button" onclick="newBookmark('${portletNamespace}');return false;" class="btn btn-default" style="display: none;"><spring:message code="portlet.view.addBookmark"/></button>

```

Comment:
Because we're in a `<c:if test="${!guestMode }">`, `guestMode` must be false. Therefore status quo 
`${ guestMode ? 'display: none;' : '' }` is always `''` (style). As I read it this change would make style always `display: none;`, so the add bookmark and add folder buttons would never show, which doesn't sound like the intention?

Consider instead removing the `style` attribute on these elements, since it's status quo not having any effect.